common-healing.lic

```diff
@@ -202,7 +202,7 @@ module DRCH
 
   def bind_wound(part, person = 'my')
     snap = [DRC.left_hand, DRC.right_hand]
-    if 'You skillfully remove' == DRC.bput("tend #{person} #{part}", 'You work carefully at tending', 'That area has already been tended to', 'That area is not bleeding', 'You fumble', 'You \w+ remove', 'You (carelessly|foolishly)', 'You deftly remove')
+    if /You \w+ remove/ =~ DRC.bput("tend #{person} #{part}", 'You work carefully at tending', 'That area has already been tended to', 'That area is not bleeding', 'You fumble', 'You \w+ remove', 'You (carelessly|foolishly)')
```

Comment:
Ok, cool. I wasn't sure what the range of matches could be, like if there were a "You foolishly invest your life savings in penny stocks"